components/profileCard/index.js

```diff
@@ -9,19 +10,41 @@ const Listitem = styled.li`
   width: 90%;
   height: 4rem;
   margin-top: 1rem;
-
   padding: 0.7rem;
+  position: relative;
 `;
 const StyledText = styled.p`
   margin: 1px;
 `;
 
+const ImageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  right: 20px;
+  gap: 5px;
+`;
+
 export default function ProfileCard({ name, email }) {
   return (
     <>
```

Comment:
It is not necessary to use an empty fragment (<>...</>) around the Listitem. You can simply return the Listitem itself.

Reply:
i need it because i got a form behind the listitem